package/gluon-ebtables-net-rules/files/lib/gluon/ebtables/110-net-allow-ipv6

```diff
@@ -0,0 +1,9 @@
+siteConfig = require("gluon.site_config")
+
+rule ('NET_SRC -p IPv6 --ip6-src fe80::/10 -j RETURN')
+rule ('NET_SRC -p IPv6 --ip6-dst ff00::/8 -j RETURN')
```

Comment:
It seems, that there are the following possible solutions:

- `rule ('NET_SRC -p IPv6 --ip6-src ::/128 -j RETURN')`
- `rule ('NET_SRC -p IPv6 --ip6-src ::/128 --ip-dst ff00::/8 -j RETURN')`
- `rule ('NET_SRC -p IPv6 --ip6-src ::/128 --ip6-protocol ipv6-icmp -j RETURN')`

Just allowing each possible icmp-type individually seemes unreasonable.